dialogue.js

```diff
@@ -461,8 +461,8 @@ window.dialogue_spiritBear=function(player,items){
spiritBear');},"Terrific!",function(){addReward([['honey',12500000000],['glitter',3],['starJelly',3],['fieldDice',15],['microConverter',5],['magicBean',5],['whirligig',10],['jellyBeans',3]])},"This is the 2nd last quest in my questline!",function(){player.addQuest('Our Sky',[['whitePollen',9000000000],['bluePollen',5000000000],['pollenFromPineTreeForest',1000000000],['pollenFromCactusField',600000000],['pollenFromRoseField',500000000],['pollenFromStumpField',100000000],['flames',75],['timesUsingTheSlingshot',100],['cloudVialTokens',15],['gumdropsTokens',25]],'spiritBear');},"Nice! Get ready for the finale quest!",function(){addReward([['honey',17500000000],['glitter',3],['starJelly',5],['loadedDice',1],['microConverter',5],['glue',5],['superSmoothie',1]])},"This is the ultimate finale of the petal questlines! It'll be the hardest quest yet!",function(){player.addQuest('My Starlight',[['whitePollen',15000000000],['redPollen',15000000000],['bluePollen',15000000000],['goo',1000000000],['pollenFromMountainTopField',5000000000],['pollenFromRoseField',3000000000],['pollenFromSpiderField',1000000000],['bubbles',150],['starJellyTokens',5],['tokensFromSprouts',500],['neonberry',25],['tokensFromWildWindyBee',300],['starJellyToTheWindShrine',15]],'spiritBear');},"Nice job! You did it! Here's your last Sprit Petal and a Star Egg!",function(){addReward([['honey',25000000000],['glitter',15],['spiritPetal',1],['starEgg',1],['starJelly',10],['superSmoothie',1],['fieldDice',5],['tropicalDrink',15],['cloudVial',3]])},"Now it's time for the 5 extra quests!","There will be 4 very hard but rewarding quests, and a star-related quest rewarding another Star Egg at the end!","(i had so many fun references to put so i had to make extra quests lol)","It begins now!",function(){player.addQuest('Hear You in the Breeze',[['bluePollen',20000000000],['whitePollen',20000000450],['whitePollen',20000000450],['pollenFromSunflowerField',750000000],['pollenFromDandelionField',500000000],['hasteTokens',450],['timesUsingTheYellowCannon',45],['fieldDice',10],['timesUsingTheSlingshot',35],['pollenFromMountainTopField',2500000000],['bubbles',250],['pollenFromPineTreeForest',1500000000],['whirligig',5],['giantAnt',45],['melodyTokens',45],['pollenFromStumpField',500000000],['glue',5]],'spiritBear');},"Nice! That's your first advanced quest done!",function(){addReward([['honey',3500000000],['glitter',3],['cloudVial',1],['jellyBeans',5],['tropicalDrink',3],['glue',3]])},"From now on, I'll give you lots of Star Jellies!","Start this next quest:",function(){player.addQuest('Doubled Winds',[['redPollen',4400000440],['pollenFromPineTreeForest',4400000024],['redExtract',2],['magicBeanTokens',4],['cloudVialTokens',24],['timesUsingTheRedCannon',24],['tokensFromWildWindyBee',440],['glueToTheWindShrine',3],['flames',33],['loadedDice',1]],'spiritBear');},"Cool!",function(){addReward([['honey',50000000000],['glitter',5],['starJelly',3],['enzymes',5],['smoothDice',2],['softWax',5],['spuerSmoothie',1]])},
```

Comment:
Removed 4 moon charms in this large block of text. Using an unminifer wasn't very useful here, but the simple VSCode online editor provided via GitHub got the job done well.